### PR TITLE
Task_one_completed

### PR DESCRIPTION
**Pull request by p4ndish**
Added custom tasks "/help", "/chat".
bot at the readme

### CODE REVIEW

File: Part_1/Dagim/bot/bot_instance.py
Comment (yohans-kasaw) [inline Part_1/Dagim/bot/bot_instance.py:1]: more commands :smile:

**Review by yohans-kasaw — APPROVED** [review]

**Comment by yohans-kasaw** [comment]
LGTM

**Review by yohans-kasaw — APPROVED** [review]
LGTM

File: Part_1/Dagim/bot/bot_instance.py
Comment (p4ndish) [reply Part_1/Dagim/bot/bot_instance.py:1]: soon... i will add it on the next task :)

**Pull request merged**